iocore/net/SSLUtils.cc

```diff
@@ -2055,8 +2055,12 @@ SSLParseCertificateConfiguration(const SSLConfigParams *params, SSLCertLookup *l
 
   // load the global ticket key for later use
   REC_ReadConfigStringAlloc(ticket_key_filename, "proxy.config.ssl.server.ticket_key.filename");
-  ats_scoped_str ticket_key_path(Layout::relative_to(params->serverCertPathOnly, ticket_key_filename));
-  global_default_keyblock = ssl_create_ticket_keyblock(ticket_key_path); // this function just returns a keyblock
+  if (ticket_key_filename && strcmp(ticket_key_filename, "\"\"") != 0) {
+    ats_scoped_str ticket_key_path(Layout::relative_to(params->serverCertPathOnly, ticket_key_filename));
+    global_default_keyblock = ssl_create_ticket_keyblock(ticket_key_path); // this function just returns a keyblock
+  } else {
+    global_default_keyblock = ssl_create_ticket_keyblock(NULL);
+  }
```

Comment:
That was a work around to get around the keyblock issue.  Removed it now that I rebased.